docs/global/modules/core/pages/howto/mps-model-server-plugin.adoc

```diff
@@ -39,4 +39,9 @@ starting modelix server
 The `model-server` will run by default on port `48305`.
 You can check the health status over at http://127.0.0.1:48305/health[^].
 
-If you want to find out how connect a client and send queries to the plugin, have a look at the xref:howto/modelql.adoc[ModelQL documentation] or the xref:core:reference/component-mps-model-server-plugin.adoc[component reference].
+If you want to find out how connect a client and send queries to the plugin, have a look at the usage docs for the
+
+* xref:howto/modelql.adoc[ModelQL],
+* xref:howto/usage-light-model-client.adoc[light-model-client],
+* xref:howto/usage-model-client-v2.adoc[model-client v2], or
```

Comment:
model-client v2 does not work with the `mps-model-server-plugin`.
The `mps-model-server-plugin` only runs the "ligth model server".